stream-platform/src/main/java/io/camunda/zeebe/stream/impl/StreamProcessor.java

```diff
@@ -264,7 +264,7 @@ private boolean shouldProcessNext() {
 
   private void tearDown() {
     processorActorService.close();
-    asyncActor.closeAsync();
```

Comment:
Not sure whether this will work, since we are not allowed to Block in actors.

Try the runOnCompletionBlocking Phase method

Reply:
I think I found a correct way to do this, check out 9f900e3